### PR TITLE
Update CONNECT-IP to match the change to HTTP datagrams

### PR DESCRIPTION
**Pull request by tfpauly**
Aligning CONNECT-IP with the changes in HTTP datagrams and CONNECT-UDP.

### CODE REVIEW

**Review by mirjak — APPROVED** [review]

**Review by DavidSchinazi — APPROVED** [review]
Overall looks great, thanks for writing this!
Approved modulo 2 small comments, (and also please don't merge until the corresponding HTTP Datagrams PR is merged)

File: draft-ietf-masque-connect-ip.md
Comment (tfpauly) [reply draft-ietf-masque-connect-ip.md:121]: Yeah, SHOULD makes sense

File: draft-ietf-masque-connect-ip.md
Comment (tfpauly) [reply draft-ietf-masque-connect-ip.md:351]: It does work!

https://ietf-wg-masque.github.io/draft-ietf-masque-connect-ip/tfp/datagram-update/draft-ietf-masque-connect-ip.html

File: draft-ietf-masque-connect-ip.md
Comment (tfpauly) [reply draft-ietf-masque-connect-ip.md:121]:
```suggestion
The client SHOULD also include the "Capsule-Protocol" header with a value of "?1"
```

**Comment by tfpauly** [comment]
@DavidSchinazi when do you plan to merge the other PRs?

File: draft-ietf-masque-connect-ip.md
Comment (DavidSchinazi) [reply draft-ietf-masque-connect-ip.md:351]: Sweet! I guess I need to update my local software :P

**Comment by DavidSchinazi** [comment]
@tfpauly I plan on merging the other PRs as soon as we have WG consensus, which (assuming no objections) should happen at the end of the month (see [email from chairs](https://mailarchive.ietf.org/arch/msg/masque/2skE5g0wH4EMAzPm6m3lH3Fo_cc/))

**Pull request merged**
